web/app/themes/ppj/functions.php

```diff
@@ -40,7 +40,17 @@ function registerImagesSizes()
     add_image_size( 'header-landscape-home', 1024, 444, true );
 
     add_image_size( 'header-large',      1440, 560, true );
+
     add_image_size( 'header-large-home', 1440, 624, true );
+
+    add_image_size('16:9x20', 320, 180, true);
+    add_image_size('16:9x30', 480, 270, true);
+    add_image_size('16:9x40', 640, 360, true);
```

Comment:
What's the naming convention behind these sizes? For example in `16:9x20` – what does the `20` bit mean?

Reply:
Added documentation.